frontend/ui/opdsparser.lua

```diff
@@ -73,6 +72,8 @@ function OPDSParser:createFlatXTable(xlex, curr_element)
 end
 
 function OPDSParser:parse(text)
+    -- Murder Calibre's whole "content" block, because it's not XML, it's XHTML, and luxl doesn't like it one bit...
```

Comment:
XHTML is definitely XML. If it's not well-formed XML it's not XHTML. ;-)

Reply:
True, what I meant by that is that it's not an HTML parser, so it gets thoroughly thrown by the HTML bits ;).

Reply:
At a guess, I'd say, among other things, something like `<em/>` vs. `</em>`...

Reply:
A self-closing em would be useless but well-formed. :-p